draft-boucadair-add-deployment-considerations.txt

```diff
@@ -313,27 +313,27 @@ Internet-Draft    Discovery of Encrypted DNS Resolvers    September 2022
    For example, DoH-capable DNS clients establish the DoH session with
    the discovered DoH server.
 
-   The DNS client discovers whether the network-designated DNS resolver
-   supports a given encrypted DNS scheme (e.g., DoT or DoH) by using the
-   "alpn" service parameter (Section 3.1.5 of [I-D.ietf-add-dnr]).
+   When the CPE supports DNR, the DNS client discovers whether the
+   network-designated DNS resolver supports a given encrypted DNS scheme
+   (e.g., DoT or DoH) by using the "alpn" service parameter
+   (Section 3.1.5 of [I-D.ietf-add-dnr]).  Otherwise, the DNS client
+   uses DDR with the Do53 resolver advertised by the CPE and upgrades to
+   encrypted DNS if that succeeds.  Otherwise, the DNS client may fall
+   back to using unencrypted DNS to the IP address advertised by the CPE
+   or use some other configuration it has.
 
+   DNR is attempted first because it requires fewer round trips to any
+   network peer because all of the necessary information to use
+   encrypted DNS is presented directly by the CPE.  DDR requires the DNS
+   client to receive configuration from the CPE and then further query
+   for encrypted DNS support from the DNS resolver before any encrypted
```

Comment:
You may want to say "Do53 resolver configuration".

Reply:
Great idea. I'll issue a commit with this change shortly.